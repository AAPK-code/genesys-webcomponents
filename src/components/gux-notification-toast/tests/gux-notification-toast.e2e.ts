import { newE2EPage } from '@stencil/core/testing';

describe('gux-notification-toast', () => {
  it('renders', async () => {
    const page = await newE2EPage();

    await page.setContent('<gux-notification-toast></gux-notification-toast>');
    const element = await page.find('gux-notification-toast');
    expect(element).toHaveClass('hydrated');
  });
});
