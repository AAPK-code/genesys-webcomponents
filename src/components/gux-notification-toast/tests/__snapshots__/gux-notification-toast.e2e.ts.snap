// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-notification-toast #render should render alert notification toast 1`] = `"<gux-notification-toast lang=\\"en\\" accent=\\"alert\\" class=\\"hydrated\\"><!----> <div class=\\"icon alert\\"><gux-icon slot=\\"icon\\" icon-name=\\"add-user\\" decorative=\\"\\" class=\\"hydrated\\"><svg aria-hidden=\\"\\" viewBox=\\"0 0 32 32\\"><use xlink:href=\\"#gux-icon-add-user\\"></use></svg></gux-icon></div><div class=\\"content\\"><div class=\\"title\\"><div slot=\\"title\\">Title</div></div><div class=\\"message\\"><div slot=\\"message\\">This is the message</div></div></div><button class=\\"dismiss-button\\" title=\\"Dismiss\\"><gux-icon class=\\"hydrated\\"><svg aria-label=\\"Dismiss\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#gux-icon-ic-close\\"></use></svg></gux-icon></button></gux-notification-toast>"`;

exports[`gux-notification-toast #render should render neutral notification toast 1`] = `"<gux-notification-toast lang=\\"en\\" accent=\\"neutral\\" class=\\"hydrated\\"><!----> <div class=\\"icon neutral\\"><gux-icon slot=\\"icon\\" icon-name=\\"add-user\\" decorative=\\"\\" class=\\"hydrated\\"><svg aria-hidden=\\"\\" viewBox=\\"0 0 32 32\\"><use xlink:href=\\"#gux-icon-add-user\\"></use></svg></gux-icon></div><div class=\\"content\\"><div class=\\"title\\"><div slot=\\"title\\">Title</div></div><div class=\\"message\\"><div slot=\\"message\\">This is the message</div></div></div><button class=\\"dismiss-button\\" title=\\"Dismiss\\"><gux-icon class=\\"hydrated\\"><svg aria-label=\\"Dismiss\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#gux-icon-ic-close\\"></use></svg></gux-icon></button></gux-notification-toast>"`;

exports[`gux-notification-toast #render should render positive notification toast 1`] = `"<gux-notification-toast lang=\\"en\\" accent=\\"positive\\" class=\\"hydrated\\"><!----> <div class=\\"icon positive\\"><gux-icon slot=\\"icon\\" icon-name=\\"add-user\\" decorative=\\"\\" class=\\"hydrated\\"><svg aria-hidden=\\"\\" viewBox=\\"0 0 32 32\\"><use xlink:href=\\"#gux-icon-add-user\\"></use></svg></gux-icon></div><div class=\\"content\\"><div class=\\"title\\"><div slot=\\"title\\">Title</div></div><div class=\\"message\\"><div slot=\\"message\\">This is the message</div></div></div><button class=\\"dismiss-button\\" title=\\"Dismiss\\"><gux-icon class=\\"hydrated\\"><svg aria-label=\\"Dismiss\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#gux-icon-ic-close\\"></use></svg></gux-icon></button></gux-notification-toast>"`;

exports[`gux-notification-toast #render should render warning notification toast 1`] = `"<gux-notification-toast lang=\\"en\\" accent=\\"warning\\" class=\\"hydrated\\"><!----> <div class=\\"icon warning\\"><gux-icon slot=\\"icon\\" icon-name=\\"add-user\\" decorative=\\"\\" class=\\"hydrated\\"><svg aria-hidden=\\"\\" viewBox=\\"0 0 32 32\\"><use xlink:href=\\"#gux-icon-add-user\\"></use></svg></gux-icon></div><div class=\\"content\\"><div class=\\"title\\"><div slot=\\"title\\">Title</div></div><div class=\\"message\\"><div slot=\\"message\\">This is the message</div></div></div><button class=\\"dismiss-button\\" title=\\"Dismiss\\"><gux-icon class=\\"hydrated\\"><svg aria-label=\\"Dismiss\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#gux-icon-ic-close\\"></use></svg></gux-icon></button></gux-notification-toast>"`;
